src/moments.jl

```diff
@@ -223,7 +223,7 @@ end
 ##### General central moment
 function _moment2(v::RealArray, m::Real; corrected=false)
     n = length(v)
-    s = 0.0
+    s = zero(eltype(v))
```

Comment:
```suggestion
    s = zero(promote_type(eltype(v), typeof(m))
```

Reply:
I think this applies for all of changes here.

Reply:
Everybody wants a promotion...